### PR TITLE
feat: add zksync constants

### PR DESCRIPTION
**Pull request by dohaki**
Add zkSync chain ids and token addresses

### CODE REVIEW

File: utils/constants.ts [hunk 1]
```diff
@@ -82,6 +84,8 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.ARBITRUM_GOERLI]: "0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3",
       [CHAIN_IDs.GOERLI]: "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
       [CHAIN_IDs.SEPOLIA]: "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
+      [CHAIN_IDs.ZK_SYNC]: "0x8Ebe4A94740515945ad826238Fc4D56c6B8b0e60",
```
Comment (nicholaspai) [inline utils/constants.ts:87]: Is this right?

File: utils/constants.ts [hunk 2]
```diff
@@ -82,6 +84,8 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.ARBITRUM_GOERLI]: "0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3",
       [CHAIN_IDs.GOERLI]: "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
       [CHAIN_IDs.SEPOLIA]: "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
+      [CHAIN_IDs.ZK_SYNC]: "0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91",
```
Comment (nicholaspai) [inline utils/constants.ts:87]: Should this be the native eth or weth address?

**Review by nicholaspai — COMMENTED** [review]
Just curious, how did you verify these addresses?

File: utils/constants.ts (hunk 2 above)
Comment (dohaki) [reply utils/constants.ts:87]: Based on the other addresses (for Arbitrum and Optimism), these should be WETH addresses.

**Comment by dohaki** [comment]
> Just curious, how did you verify these addresses?

I first looked for them here https://explorer.zksync.io/tokenlist and for others I tried to find them here https://www.geckoterminal.com/zksync/pools

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**